tools/test.py

```diff
@@ -237,8 +237,9 @@ def main():
             model.cuda(),
             device_ids=[torch.cuda.current_device()],
             broadcast_buffers=False)
-        outputs = multi_gpu_test(model, data_loader, args.tmpdir,
-                                 args.gpu_collect)
+        outputs = multi_gpu_test(
+            model, data_loader, args.tmpdir, args.gpu_collect
+            or cfg.evaluation.gpu_collect)
```

Comment:
what if the cfg.evaluation.gpu_collect is not set? Will it cause error？

Reply:
Yes, it will. I'll fix it soon

Reply:
We can use `args.gpu_collect or cfg.evaluation.get('gpu_collect', False)` here.